api/conf/settings/base.py

```diff
@@ -60,8 +60,6 @@
     },
 ]
 
-WSGI_APPLICATION = "conf.wsgi.application"
-
```

Comment:
I had a faint recollection that `ASGI_APPLICATION` was needed here. Turns out it was just a requirement for [Daphne](https://docs.djangoproject.com/en/4.2/howto/deployment/asgi/daphne/).